lib/LambdaHelper.js

```diff
@@ -16,6 +25,42 @@ var _getFunctionHash  = function (func) {
     // TODO : add config in hash
 };
 
+var _getModuleHash = function (modulePath, handlerName, deps, configs) {
+    var configsAsStr = JSON.stringify(configs),
+        identifier = [MODULE_HASH_PREFIX, modulePath, handlerName, deps, configsAsStr].join('_');
+
+    return md5(identifier); // TODO : add config in hash?
+};
+
+var _loadModule = function(moduleName) {
+    return require(moduleName); // TODO throw better errors if require fails
+};
+
+var _getDependencyFolder = function(dep) {
+    var depFolder = path.dirname(require.resolve(dep)) + path.sep;
+    var subFolders = depFolder.split(path.sep);
+    var folders = subFolders.splice(0, subFolders.indexOf('node_modules') + 2);
```

Comment:
oups yeah that isn't clear. You need +1 because indexOf returns a zero-based index and an other +1 because you want the next folder